src/components/radios/default.md.njk

```diff
@@ -0,0 +1,25 @@
+---
+layout: layout-example.njk
+---
+
+{% from 'radio/macro.njk' import govukRadio %}
+
+{{ govukRadio({
+  "idPrefix": "example",
+  "name": "example",
+  "fieldset": {
+    "legendHtml": "<h1 class=\govuk-heading-xl\>Have you changed your name?</h1>",
```

Comment:
I would've probably used a mix of single and double quotes `'legendHtml': '<h1 class="govuk-heading-xl">Have you changed your name?</h1>'`, but it's just a personal option.